### ISSUE TITLE
Setting up the /Users NFS share requires -network and -mask

### ISSUE THREAD

**Issue by marceldegraaf** [IFTTT/dash#8]
I'm trying out Dash on my Mac (OS/X El Capitan 10.11.1) and I noticed that setting up the NFS share in `bootsync.sh` doesn't work without the `-network` and `-mask` flags.

This is the `/etc/exports` line that Dash uses: `"/Users\" 192.168.99.100 -alldirs -mapall=0:80`
However, that configuration yields an error in the Virtualbox VM for me:

```
mount.nfs: mount(2): Permission denied
mount.nfs: access denied by server while mounting 192.168.99.1:/Users
```

To get it working, I updated the line in `/etc/exports` to: `/Users -alldirs -mapall=0:80 -network 192.168.0.0 -mask 255.255.0.0`, to explicitly allow access to the Virtualbox subnet.

Is this a know issue, or am I missing something else?

**Comment by silvamerica** [comment]
I haven't run into that issue before, but it's possible that something changed in El Capitan. Did your Mac always have 10.11 on it, or is it an upgrade?

**Comment by marceldegraaf** [comment]
No, it was upgraded from Mavericks to Yosemite, and now to El Capitan.
